sw/airborne/arch/chibios/modules/actuators/esc_dshot.c

```diff
@@ -0,0 +1,482 @@
+/*
+ * Copyright (C) 2018 Alexandre Bustico <alexandre.bustico@enac.fr>
+ *                    Gautier Hattenberger <gautier.hattenberger@enac.fr>
+ *
+ * This file is part of paparazzi
+ *
+ * paparazzi is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2, or (at your option)
+ * any later version.
+ *
+ * paparazzi is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with paparazzi; see the file COPYING.  If not, see
+ * <http://www.gnu.org/licenses/>.
+ */
+
+#include "modules/actuators/esc_dshot.h"
+#include <stdnoreturn.h>
+#include <math.h>
+#include <string.h>
+
+/*
```

Comment:
same as for `hal_stm32_dma`, I'm not the author nor maintainer of these files (used in other internal ENAC project), so I tried to avoid unnecessary modifications to the original files as it gives me even more work to merge bug fix and evolution.